### ISSUE TITLE
error after apparently successful install

### ISSUE THREAD

**Issue by fredzannarbor** [hugorut/neural-cli#1]
Ubuntu 16.04, Python 3.5.2

```
Collecting neuralcli
Requirement already satisfied (use --upgrade to upgrade): numpy in /usr/local/lib/python3.5/dist-packages (from neuralcli)
Requirement already satisfied (use --upgrade to upgrade): click in /usr/local/lib/python3.5/dist-packages (from neuralcli)
Requirement already satisfied (use --upgrade to upgrade): sklearn in /usr/local/lib/python3.5/dist-packages (from neuralcli)
Requirement already satisfied (use --upgrade to upgrade): scipy in /usr/local/lib/python3.5/dist-packages (from neuralcli)
Requirement already satisfied (use --upgrade to upgrade): matplotlib in /usr/lib/python3/dist-packages (from neuralcli)
Requirement already satisfied (use --upgrade to upgrade): scikit-learn in /usr/local/lib/python3.5/dist-packages (from sklearn->neuralcli)
Installing collected packages: neuralcli
Successfully installed neuralcli-1.2

```
```
fred@fred-XPS-L322X:~$ command -v neuralcli
/usr/local/bin/neuralcli
fred@fred-XPS-L322X:~$ which neuralcli
/usr/local/bin/neuralcli
fred@fred-XPS-L322X:~$ neuralcli
Traceback (most recent call last):
  File "/usr/local/bin/neuralcli", line 7, in <module>
    from neuralcli.cli import main
  File "/usr/local/lib/python3.5/dist-packages/neuralcli/cli.py", line 65
    print nn.predict(x)

```

**Comment by hugorut** [comment]
Hey thanks for pulling this up. This is a bug with the package compatibility with python 3.5. I will work on a fix tonight.

**Comment by hugorut** [comment]
@fredzannarbor this issue was fixed with version `1.4` let me know if you still run into problems
